frontend/src/components/assessments/assessment-creation/DisplayQuestion.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {
+  Box,
+  Checkbox,
+  CheckboxGroup,
+  Input,
+  Radio,
+  RadioGroup,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+
+import { QuestionComponentResponse } from "../../../APIClients/types/TestClientTypes";
+import { QuestionElementType } from "../../../types/QuestionTypes";
+
+interface DisplayQuestionProps {
+  questionComponents: QuestionComponentResponse[];
+}
+
+const DisplayQuestion = ({
+  questionComponents,
+}: DisplayQuestionProps): React.ReactElement => {
+  const stackProps = {
+    alignItems: "left",
+    gap: "1",
+    paddingRight: "4",
+    paddingTop: "2",
+    ml: "5",
+  };
+
+  return (
+    <>
+      {questionComponents.map((question, i) => {
+        return (
+          <Box key={i} my={10}>
+            {(() => {
+              switch (question.type) {
+                case QuestionElementType.QUESTION_TEXT:
+                  return (
+                    <Text textStyle="subtitle1">
+                      {question.metadata.questionText}
```

Comment:
to fix the error that you're having you need to cast it to the correct type before trying to access the type's fields!

```suggestion
                     {(question.metadata as QuestionTextMetadata).questionText}
```

Reply:
OHHH thank youu